modules/core/src/lib/attribute/data-column.js

```diff
@@ -276,7 +276,11 @@ export default class DataColumn {
       type: this.defaultType,
       copy
     });
-    this.value = value;
+
+    // Hack - Using typed array manager props to let app pass this prop here
+    if (!typedArrayManager.props._discardCPUArrays) {
```

Comment:
How will this work? The auto-updater expects `value` to be available.

Reply:
Yes good question now that I review it myself. I have removed this change from the PR until I have time to give it another look. 

FWIW:
- I'd like to add stats support to the typed array manager so we can track how much is actually being allocated.
- Then make another pass at disabling the saving of CPU attributes.